public/index.html

```diff
@@ -134,11 +153,6 @@ <h3>Anna Geiduschek</h3>
 									<h3>Ben Packer</h3>
 									<p>Data Scientist, Recidiviz. Data Scientist, Google. </p>
```

Comment:
I've noticed a spacing issue with `featurettes` where if a row of contributors has one-line descriptions for each column, this causes the face bubbles for those contributors to scrape the bottom of the cell boundary. You can see this on the row with Anna and Ben, for example.